README.md

```diff
@@ -339,19 +341,27 @@ All the below functions return a Promise which gets resolved with the result.
 
 ## Data Unions
 
-This library provides functions for working with Data Unions. To get a DataUnion instance, call `client.getDataUnion(address)`. To deploy a new DataUnion, call `deployDataUnion(options)`
+This library provides functions for working with Data Unions.
 
-See `/docs` folder in this repo for Data Union `options` parameters. These docs can be rebuilt locally via:
+To deploy a new DataUnion:
```

Comment:
Is this mixed up with 
**_To get an existing (previously deployed) `DataUnion` instance:_**

Reply:
good catch